lib/msal-browser/docs/events.md

```diff
@@ -0,0 +1,48 @@
+# Events
+
+MSAL.js 2.0 provides event APIs that are available to users of our core library and wrapper libraries. These events are related to auth and what MSAL is doing, and can be used in applications to update UI, show error messages, and so on.
+
+## What Events look like
+```javascript
+export type EventMessage = {
+    eventType: EventType;
+    interactionType: InteractionType | null;
+    payload: EventPayload;
+    error: EventError;
+    timestamp: number;
+};
+```
+
+## How to use the Event API
+Msal-browser exports the `addEventCallback` function which takes in a callback function and can be used to process emitted events. Events are emitted in msal-browser according to the list below.
+
+Here is an example of how you could consume the emitted events in your application:
+```javascript
+msalInstance.addEventCallback((message: EventMessage) => {
+    // Update UI or interact with EventMessage here
+    if (message.eventType === EventType.LOGIN_SUCCESS) {
+        console.log(message.payload);
+});
+```
+
+## List of Event Types
+
+This the list of events currently emitted by msal-browser.
+
+| Event Type  | Description  | Interaction Type  | Payload   | Error |
+|------------ |:----------: |:-----------: |:------------: | :---------:|
+| `LOGIN_START` | LoginPopup or loginRedirect are called | `Popup` or `Redirect` | PopupRequest or RedirectRequest | 
```

Comment:
For readability during editing, its often nice to line up the columns.

Reply:
Makes sense. Done.